packages/maskbook/src/utils/native-rpc/types.ts

```diff
@@ -1,9 +1,13 @@
+import type { PersonaWithProviders } from '../../database'
 /**
  * JSON RPC calls that can be called from the native side.
  * JS = server, native = client
  */
 export interface WebviewAPIs {
     web_echo<T>(arg: T): Promise<T>
+    getUrl(path: string): string
```

Comment:
Please mark it as Promise<string>

Reply:
`browser.runtime.getURL(path)` is not an async function, should still mark it as Promise?

Reply:
Yes. It's always async because it is RPC. And I suggest native side to make things more clear. Expose a `getDashboardURL(): Promise<string>` instead of exposing the underlying URL detail. If so, refactoring on the URL will cause a breaking change in the native side which is unwanted.